### DISCUSSION TITLE
组个code-server，使用国内节点并且开启自动https之后，进不去 code-server

### DISCUSSION THREAD

**Discussion by Huoyucat** [natfrp/wiki#327] [使用问题]
节点设置
![image](https://github.com/natfrp/wiki/assets/74894658/a3373095-44f9-4365-bd91-2bd91572353c)

这是code-server的配置
![屏幕截图 2023-05-19 153044](https://github.com/natfrp/wiki/assets/74894658/f5b02d19-690d-41cc-a1da-ca45eafad805)
用https://+frp的地址进去code-server之后
![屏幕截图 2023-05-19 153127](https://github.com/natfrp/wiki/assets/74894658/71d01d8c-d530-4553-936d-eec972361d70)

改用海外节点就可以进去，想问一下怎么解决这个问题

**Comment by B1INGO** [comment]
我刚好迷迷糊糊解决了这个问题。
使用本地自签证书。
生成证书：sudo openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout $HOME/mykey.key -out $HOME/mycert.crt
停止code-server：sudo systemctl stop code-server@$USER
使用证书启动code-server：code-server --cert mycert.crt --cert-key mykey.key

**Comment by B1INGO** [comment]
对了我使用的是国内节点的tcp+自动https

**Reply by Huoyucat** [reply]
想问一下你 code-server的config文件是怎么设置的

**Reply by Huoyucat** [reply]
我的是wsl里面的code-server，会有其他影响吗
